### PR TITLE
Adding functions to the __init__.py files

### PR DESCRIPTION
**Pull request by AdamTheisen**
We were missing some of the functions in the __init__.py files.  We might want to do an inventory to make sure we aren't missing anything else.

### CODE REVIEW

File: act/qc/__init__.py [hunk 1]
```diff
@@ -25,9 +25,11 @@
     ],
     submod_attrs={
         'arm': ['add_dqr_to_qc'],
-        'qcfilter': ['QCFilter'],
```
Comment (zssherman) [inline act/qc/__init__.py:28]: do we still need to specify QCFilter and QCtests

Comment (AdamTheisen) [reply act/qc/__init__.py:28]: @zssherman  good question.  I took out QCTests from the modules but we need to keep QCFilter in there because there are 3 functions outside the class that we need to load up that we are not currently doing so.

**Comment by AdamTheisen** [comment]
I am finding a few more issues with the documentation so working through things.  Hold on any approvals/merging

**Comment by AdamTheisen** [comment]
@zssherman  I think I have all the issues solved.  @mgrover1 would you want to take a look as well?

**Review by mgrover1 — APPROVED** [review]
This looks good - I can put together a list of all of the modules and associated submodules to make sure we don't miss anything.

**Comment by AdamTheisen** [comment]
Hold for one sec.  When removing the QCFilter, it no longer is included in the documentation.  Let me look into this

File: act/qc/__init__.py [hunk 2]
```diff
@@ -19,15 +19,16 @@
         'bsrn_tests',
         'comparison_tests',
         'qcfilter',
-        'qctests',
```
Comment (mgrover1) [inline act/qc/__init__.py:22]: Why did you remove qctests here?

File: act/qc/__init__.py [hunk 3]
```diff
@@ -19,15 +19,16 @@
...
         'radiometer_tests',
         'sp2',
     ],
     submod_attrs={
         'arm': ['add_dqr_to_qc'],
-        'qcfilter': ['QCFilter'],
-        'qctests': ['QCTests'],
```
Comment (mgrover1) [inline act/qc/__init__.py:29]: same here

File: act/qc/__init__.py (hunk 2 above)
Comment (AdamTheisen) [reply act/qc/__init__.py:22]: @mgrover1 we are doing direct imports of QCTests so it should be available.  Do we need to keep it in the lazy_loader section if we are already importing it?

```
from .qcfilter import QCFilter
from .qctests import QCTests
```

File: act/qc/__init__.py (hunk 3 above)
Comment (AdamTheisen) [reply act/qc/__init__.py:29]: @mgrover1 see previous response.  If we are importing it directly at the start of the __init__ do we need it in the lazy_loader modules?

File: act/qc/__init__.py (hunk 2 above)
Comment (mgrover1) [reply act/qc/__init__.py:22]: Good point - it looks like it is showing up in the built documentation in the github action log

**Comment by mgrover1** [comment]
@AdamTheisen does your main __init__.py file in `act/__init__,py` include this?

https://github.com/ARM-DOE/ACT/blob/main/act/__init__.py#L14

**Comment by AdamTheisen** [comment]
@mgrover1 yes it does.  Looks like we have it in both the act level and qc level.

https://github.com/ARM-DOE/ACT/blob/829c5a1fffcd5b690ccbe493fbdcc62b52b8bd10/act/qc/__init__.py#L11

**Comment by mgrover1** [comment]
@AdamTheisen and the docs aren't showing this on your local act doc build?

**Comment by AdamTheisen** [comment]
@mgrover1 I'm was just going off of what was already in there.  I do see some weirdness though in the QCFilter documentation in that is has all the QCTest functions as well. https://arm-doe.github.io/ACT/search.html?q=QCFilter

**Comment by mgrover1** [comment]
@AdamTheisen is this building locally

> @AdamTheisen and the docs aren't showing this on your local act doc build?

**Comment by AdamTheisen** [comment]
@mgrover1 this is building locally now.  I think I got all the issues resolved that were impacting my builds.

**Comment by mgrover1** [comment]
Looks good to me!

**Comment by zssherman** [comment]
@AdamTheisen Good to go then? I can merge this and get a release out.

**Review by zssherman — APPROVED** [review]

**Comment by AdamTheisen** [comment]
@zssherman yep, I think we're good to go here
